src/google/protobuf/compiler/subprocess.cc

```diff
@@ -126,30 +125,17 @@ void Subprocess::Start(const std::string& program, SearchMode search_mode) {
     GOOGLE_LOG(FATAL) << "GetStdHandle: " << Win32ErrorMessage(GetLastError());
   }
 
-  // get wide string version of program as the path may contain non-ascii characters
-  std::wstring wprogram;
-  if (!io::win32::strings::utf8_to_wcs(program.c_str(), &wprogram)) {
-    GOOGLE_LOG(FATAL) << "utf8_to_wcs: " << Win32ErrorMessage(GetLastError());
-  }
-
   // Invoking cmd.exe allows for '.bat' files from the path as well as '.exe'.
-  std::string command_line = "cmd.exe /c \"" + program + "\"";
-
-  // get wide string version of command line as the path may contain non-ascii characters
-  std::wstring wcommand_line;
-  if (!io::win32::strings::utf8_to_wcs(command_line.c_str(), &wcommand_line)) {
-    GOOGLE_LOG(FATAL) << "utf8_to_wcs: " << Win32ErrorMessage(GetLastError());
-  }
```

Comment:
this looks like it is reverting an important change

Reply:
I'm performing a resync which should clean this up.